### PR TITLE
Feature/display orbs in metamask

### PR DESCRIPTION
**Pull request by gilamran**
## Description

Added a simple button that lets our users view ORBS token in their metamask.

### CODE REVIEW

File: voting/client-web/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -29,6 +30,20 @@ const HeaderImpl = ({ classes, isReadOnly }) => {
     { label: t('Rewards'), url: '/reward' },
   ];
 
+  function displayInMetamask() {
+    (window as any).web3.currentProvider.sendAsync({
```
Comment (OrLavy) [inline voting/client-web/src/components/Header/Header.tsx]: Maybe put all the web3 api calls in a service ?

File: voting/client-web/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -29,6 +30,20 @@ const HeaderImpl = ({ classes, isReadOnly }) => {
...
+      method: 'metamask_watchAsset',
+      params: {
+        type: 'ERC20',
+        options: {
+          symbol: 'ORBS',
+          address: '0xff56cc6b1e6ded347aa0b7676c85ab0b3d08b0fa',
+          decimals: 18,
+          image: 'https://s2.coinmarketcap.com/static/img/coins/64x64/3835.png',
```
Comment (OrLavy) [inline voting/client-web/src/components/Header/Header.tsx]: Coded hard ?

File: voting/client-web/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -29,6 +30,20 @@ const HeaderImpl = ({ classes, isReadOnly }) => {
     { label: t('Rewards'), url: '/reward' },
   ];
 
+  function displayInMetamask() {
```
Comment (OrLavy) [inline voting/client-web/src/components/Header/Header.tsx]: Need to ask you about our work practices, do we want all effects to be redux based or smth ?

Comment (gilamran) [reply voting/client-web/src/components/Header/Header.tsx]: Hedron isn't using any state management library. Depending on the use case we can decided on the proper solution. If we'll see that Hedron is getting too complicated regarding state, we'll choose a library.

File: voting/client-web/src/components/Header/Header.tsx (hunk 1 above)
Comment (gilamran) [reply voting/client-web/src/components/Header/Header.tsx]: nice catch, fixed

File: voting/client-web/src/components/Header/Header.tsx (hunk 2 above)
Comment (gilamran) [reply voting/client-web/src/components/Header/Header.tsx]: If you're talking about the image url, I think that it's self explanatory.

**Pull request merged**